NEWS

```diff
@@ -1,3 +1,40 @@
+26-Jan-2017 IGNITION v0.21.0
+
+  Features
+
+    - Add support for networkd drop-ins
+    - Add new binary, ignition-validate, for validating Ignition configs
+    - Add overwrite field to files, directories, and links for deleting
+      preexisting items at the node's path
+    - Add options field to raid for specifying arbitrary mdadm options
+    - Add append field to files for appending to preexisting files
+    - Add support for specifying custom certificate authorities to use when
+      fetching objects over HTTPS
+
+  Changes
+
+    - Validate that partition labels don't contain colons, as sgdisk will
+      silently truncate the label
+    - Remove validate flag from Ignition
```

Comment:
Might be helpful to say what version this was introduced, to signal that the change isn't a big deal.